src/InputTypes/Identity.js

```diff
@@ -0,0 +1,26 @@
+const {getCountryDisplayName} = require('../Form/formatters')
+
+/**
+ * @implements {TooltipItemRenderer}
+ */
+class IdentityTooltipItem {
+    /** @type {IdentityObject} */
+    #data;
+    /** @param {IdentityObject} data */
+    constructor (data) {
+        this.#data = data
+    }
+    id = () => String(this.#data.id)
+    primaryText = (subtype) => {
+        if (subtype === 'addressCountryCode') {
+            return getCountryDisplayName('en', this.#data.addressCountryCode || '')
+        }
+        if (this.#data.id === 'privateAddress') {
+            return 'Generated Private Address\n' + this.#data[subtype]
```

Comment:
this was previously in the DataTooltip file, much cleaner now being alongside the rest of the logic for Identity